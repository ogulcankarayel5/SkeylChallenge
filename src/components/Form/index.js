export * from './Form'
export * from './style'
export * from '../../shared/FormInput'