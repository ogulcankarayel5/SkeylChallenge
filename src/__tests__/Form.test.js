import React from 'react'

import {Form} from '../components'

import renderer from 'react-test-renderer'
import 'jest-styled-components'



describe("Form snapshot",() => {
    
    it("should match snapshot",() => {
        const tree = renderer.create(<Form/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
}) 