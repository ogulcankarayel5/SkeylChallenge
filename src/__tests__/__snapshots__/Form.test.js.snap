// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form snapshot should match snapshot 1`] = `
.c0 {
  width: 100%;
  height: 100vh;
  display: grid;
  grid-template-columns: 6fr 1fr;
  grid-template-rows: 1fr 10fr;
  grid-template-areas: "input button " "textarea textarea";
  padding: 5rem;
}

<form
  className="c0"
/>
`;
